internal/service/workspaces/directory_test.go

```diff
@@ -1256,6 +1256,40 @@ func testAccDirectory_poolsWorkspaceCreationAD(t *testing.T) {
 	})
 }
 
+func testAccDirectory_dedicatedTenancy(t *testing.T) {
```

Comment:
Can we extend this test case to include changing the `tenancy` value?

As implemented the update logic does not currently handle changes to this argument. I don't see any `Modify` APIs in the workspace [API docs](https://docs.aws.amazon.com/workspaces/latest/api/API_Operations.html) which relate to tenancy, so I suspect the schema just needs `ForceNew` added if the update handler cannot account for changes. Either way, the additional test step will ensure proper update handling is implemented.